automation/taskcluster/decision_task_publish.py

```diff
@@ -0,0 +1,224 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+Decision task for releases
```

Comment:
Nit: maybe append a "and snapshot releases as well"?

Reply:
Good call. Fixed.